htdocs/adherents/list.php

```diff
@@ -591,7 +591,7 @@
 if ($user->rights->adherent->creer) {
 	$arrayofmassactions['close'] = img_picto('', 'close_title', 'class="pictofixedwidth"').$langs->trans("Resiliate");
 }
-if ($user->rights->adherent->supprimer) {
+if ($permissiontodelete) {
```

Comment:
It seems this introduce a bug because the variable is declared into a if so is not always declared.